README.md

```diff
@@ -51,22 +51,38 @@ Windows:
 bin\hunter_env.cmd
 ```
 
-After the environment is configured, you can build for any supported `Polly` toolchain (see `polly.py --help`) using the `bin/drishti_build.{sh,cmd}` scripts.
+After the environment is configured, you can build for any supported `Polly` toolchain (see `polly.py --help`) with a command like this:
 
-Linux/OS X:
 ```
-TOOLCHAIN=osx-10-11-hid-sections-lto
-CONFIG=Release  
-INSTALL=--install
-bin/drishti_build.sh "${TOOLCHAIN}" "${CONFIG}" "${INSTALL}"
+polly.py --toolchain ${TOOLCHAIN} --config ${CONFIG} --fwd HUNTER_CONFIGURATION_TYPES=${CONFIG} --install --verbose
```

Comment:
I guess first time user is not interested in `--install`.

Reply:
I think `--install` is important.  My sense is that the majority of users are looking for a working lib they can stick into their project.  An "Integration" section discussing Hunter vs Non-Hunter (shared lib) solutions is also needed in a follow up.